dgea/plot_dgea.py

```diff
@@ -94,16 +94,30 @@ def plot_volcano():
         
         df_plot = deseq_results.copy()
         df_plot["-log10(padj)"] = -np.log10(df_plot["padj"])
+        min_value = 1e-100
+        df_plot["p-value"] = df_plot["padj"].apply(lambda x: x if x > min_value else f"less than {min_value}")
```

Comment:
Very sleek implementation. I think we can set the `min_value` higher, since the current one would probably look like `0.00000000.....00001`. `1e-5` should be sufficient